boards/nordic/nrf9280pdk/nrf9280pdk_nrf9280-ipc_conf.dtsi

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	ipc {
+		cpusec_cpuapp_ipc: ipc-1-2 {
+			compatible = "zephyr,ipc-icmsg";
+			status = "disabled";
+			mboxes = <&cpusec_bellboard 12>,
+				 <&cpuapp_bellboard 0>;
+		};
+
+		cpusec_cpurad_ipc: ipc-1-3 {
+			compatible = "zephyr,ipc-icmsg";
```

Comment:
I thought cpusec uses ipc-icbmsg. Is this compatible correct?

Reply:
> I thought cpusec uses ipc-icbmsg.

Not currently.